src/GitHub.StartPage/StartPagePackage.cs

```diff
@@ -59,7 +57,6 @@ async Task<CodeContainer> RunAcquisition(IProgress<ServiceProgressData> download
             try
             {
                 var uiProvider = await Task.Run(() => Package.GetGlobalService(typeof(IGitHubServiceProvider)) as IGitHubServiceProvider);
-                await ShowTeamExplorerPage(uiProvider);
```

Comment:
So this was the piece that was waiting for the Team Explorer to let us know that the clone had completed?

Reply:
This was a hack to make the `IGitRepositoriesExt` service available later on when the `Clone` method attempts to retrieve it from `IGitHubServiceProvider`.

The problem was that it required two pieces to be in place before it wold work.
1. The `Team Explorer - Connext` page must be visible
2. Something must have set the `GitServiceProvider` on `IGitHubServiceProvider`

This was way too fragile and the reason the bug crept in.